recipes/ogre/1.x/test_package/ogre_main.cpp

```diff
@@ -0,0 +1,17 @@
+#include <OGRE/OgreRenderSystemCapabilities.h>
+#include <OGRE/OgreParticle.h>
+#include <OGRE/OgrePrerequisites.h>
```

Comment:
```suggestion
#include <OgreRenderSystemCapabilities.h>
#include <OgreParticle.h>
#include <OgrePrerequisites.h>
```
This should fix the error, and is similar to upstream does https://github.com/OGRECave/ogre/search?q=OgreRenderSystemCapabilities.h https://github.com/OGRECave/ogre/search?q=OgreParticle.h https://github.com/OGRECave/ogre/search?q=OgrePrerequisites.h, so it most probably matches what existing users do